Example/Sources/Helper/ColorsView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ColorsView.swift
+//  Example
+//
+//  Copyright © 2020 Bottle Rocket Studios. All rights reserved.
+//
+
+import SwiftUI
+
+struct ColorsView: View {
+
+    // MARK: - Properties
+    @State private var colors: [Color]
```

Comment:
Without an init or setting this to a default value, this struct should produce an error.

Reply:
Fixed